lib/configurators/aws/aws.go

```diff
@@ -775,7 +775,7 @@ func buildSSMDocumentCreators(ssm map[string]ssmClient, targetCfg targetConfig,
 			ssmCreator := awsSSMDocumentCreator{
 				ssm:      ssm[region],
 				Name:     matcher.SSM.DocumentName,
-				Contents: awslib.EC2DiscoverySSMDocument(proxyAddr),
+				Contents: awslib.EC2DiscoverySSMDocument(proxyAddr, false),
```

Comment:
nit: spell out what does false represent here?
```suggestion
				Contents: awslib.EC2DiscoverySSMDocument(proxyAddr, false /* insecureSkipNameRandomization */),
```

Reply:
c4392cd3b49e4c02a8034b22c0987e3b1f5f847d